docs/index.asciidoc

```diff
@@ -1,6 +1,8 @@
 = APM Go Agent Reference
 
 ifdef::env-github[]
+NOTE: For the best reading experience,
+please view this documentation at https://www.elastic.co/guide/en/apm/agent/go[elastic.co]
```

Comment:
can you please add a newline after this? otherwise the bullet points don't render on github

Reply:
Whoops! Sorry about that, totally forgot to check out what it would look like on GitHub. I've added the newline.